.storybook/readmeDecorator.js

```diff
@@ -0,0 +1,14 @@
+import { makeDecorator } from "@storybook/addons";
+
+export const addReadmeToStory = makeDecorator({
+  wrapper: (storyFn, context) => {
+    try {
+      const folderPath = context.parameters.fileName.replace(/[A-Za-z.]*$/, "").replace("./src/", "");
+      context.parameters["readme"] = {
+        // https://webpack.js.org/api/module-methods/#dynamic-expressions-in-import
+        sidebar: require(`src/${folderPath}${context.name}.story.md`),
+      };
+    } catch (e) {}
```

Comment:
Need to handle the type or message of the error, so you will be silent only for those errors, that really should be silent.

We need to be silent only for Webpack's `failed to import` errors and throw all other errors